src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1894,11 +1900,12 @@ private static IEnumerable<InstrumentationKind> ParseInstrumentationKinds(string
                 return null;
             }
 
-            if (string.IsNullOrEmpty(filePath))
+            if (RoslynString.IsNullOrWhiteSpace(filePath))
```

Comment:
:memo: this change aligns with logic in `ParseResourceDescription` (called a few lines above). The effect is to change which error is reported (see `ParseResources` test, which was updated).